modules/configuring-the-lora-training-job.adoc

```diff
@@ -0,0 +1,249 @@
+        {"Tweet text":"@KristaMariePark Thank you for your interest! If you decide to cancel, you can call Customer Care at 1-800-NYTIMES.","ID":1,"Label":2,"text_label":"no complaint","output":"### Text: @KristaMariePark Thank you for your interest! If you decide to cancel, you can call Customer Care at 1-800-NYTIMES.\n\n### Label: no complaint"},
+        {"Tweet text":"@EE On Rosneath Arial having good upload and download speeds but terrible latency 200ms. Why is this.","ID":3,"Label":1,"text_label":"complaint","output":"### Text: @EE On Rosneath Arial having good upload and download speeds but terrible latency 200ms. Why is this.\n\n### Label: complaint"},
+        {"Tweet text":"Couples wallpaper, so cute. :) #BrothersAtHome","ID":4,"Label":2,"text_label":"no complaint","output":"### Text: Couples wallpaper, so cute. :) #BrothersAtHome\n\n### Label: no complaint"},
+        {"Tweet text":"@mckelldogs This might just be me, but-- eyedrops? Artificial tears are so useful when you're sleep-deprived and sp… https:\/\/t.co\/WRtNsokblG","ID":5,"Label":2,"text_label":"no complaint","output":"### Text: @mckelldogs This might just be me, but-- eyedrops? Artificial tears are so useful when you're sleep-deprived and sp… https:\/\/t.co\/WRtNsokblG\n\n### Label: no complaint"},
+        {"Tweet text":"@Yelp can we get the exact calculations for a business rating (for example if its 4 stars but actually 4.2) or do we use a 3rd party site?","ID":6,"Label":2,"text_label":"no complaint","output":"### Text: @Yelp can we get the exact calculations for a business rating (for example if its 4 stars but actually 4.2) or do we use a 3rd party site?\n\n### Label: no complaint"},
+        {"Tweet text":"@nationalgridus I have no water and the bill is current and paid. Can you do something about this?","ID":7,"Label":1,"text_label":"complaint","output":"### Text: @nationalgridus I have no water and the bill is current and paid. Can you do something about this?\n\n### Label: complaint"},
+        {"Tweet text":"@JenniferTilly Merry Christmas to as well. You get more stunning every year ��","ID":9,"Label":2,"text_label":"no complaint","output":"### Text: @JenniferTilly Merry Christmas to as well. You get more stunning every year ��\n\n### Label: no complaint"}
+    ]
+
+----
+.. Replace the example namespace value `kfto` with the name of your project.
+.. Replace the example training data with your training data.
+.. Save your changes in the `training_data.yaml` file.
+.. Apply the configuration to create the training data:
++
+[source]
+----
+$ oc apply -f training_data.yaml
+----
+
+. Create a persistent volume claim (PVC), as follows:
+.. Create a YAML file named `trainedmodelpvc.yaml`.
+.. Add the following `PersistentVolumeClaim` object definition:
++
+[source]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: trained-model
+  namespace: kfto
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
```

Comment:
```suggestion
      storage: 50Gi
```

Reply:
Larger storage is needed as the model snapshot is stored after every epoch.
Model snapshot is roughly 3GB, 10 epochs, some buffer, so 50 GB storage should be enough.

Reply:
@sutaakar When deciding how much extra storage to add as buffer, is it a percentage of the (_ModelSize_*_NumEpochs_) value, or a minimum of _X_ GB?